pkg/scheduler/apis/config/types.go

```diff
@@ -96,6 +96,10 @@ type KubeSchedulerConfiguration struct {
 	// Extenders are the list of scheduler extenders, each holding the values of how to communicate
 	// with the extender. These extenders are shared by all scheduler profiles.
 	Extenders []Extender
+
+	// NoCacheNonLeader disables caching for non-leader candidates. This makes non-leader candidates
+	// not start watching cache. Doing so will lower memory usage with slower failover.
+	NoCacheNonLeader bool
```

Comment:
Avoid double-negative. Maybe `MaintainCacheWhenNonLeader *bool`, defaulting to true.

Reply:
Done